pep-0691.rst

```diff
@@ -214,13 +217,23 @@ Each individual file dictionary has the following keys:
   The ``hashes`` dictionary **MUST** be present, even if no hashes are available
   for the file, however it is **HIGHLY** recommended that at least one secure,
   guaranteed to be available hash is always included.
+
+  By default, any hash algorithm available via `hashlib
+  <https://docs.python.org/3/library/hashlib.html>`_ (specifically any that can
+  be passed to ``hashlib.new()`` and do not require additional parameters) can
+  be used as a key for the hashes dictionary. At least one secure algorithm from
+  ``hashlib.algorithms_guaranteed`` **SHOULD** always be included. At the time
+  of this PEP, ``sha256`` specifically is recommended.
 - ``requires-python``: An **optional** key that exposes the *Requires-Python*
   metadata field, specified in :pep:`345`. Where this is present, installer tools
   **SHOULD** ignore the download when installing to a Python version that
   doesn't satisfy the requirement.
+
+  Unlike ``data-requires-python`` in :pep:`503`, the ``requires-python`` key does not
+  require any special escaping other than anything JSON does naturally.
 - ``dist-info-metadata-available``: An **optional** key that indicates
   that metadata for this file is available, via the same location as specified in
-  :pep:`658` (`{file_url}.metadata`). Where this is present, it **MUST** be true,
+  :pep:`658` (``{file_url}.metadata``). Where this is present, it **MUST** be true,
   or a dictionary mapping a hash name to a hex encoded digest of the metadata hash.
```

Comment:
Let's add a reference to the hashes key above, noting that the contents are similar to that key but for different content?